client/app/privkey.go

```diff
@@ -98,3 +114,87 @@ func exists(file string) bool {
 	_, err := os.Stat(file)
 	return !os.IsNotExist(err)
 }
+
+// EncryptedKeyRepresentation defines an internal representation of encrypted validator key.
+type EncryptedKeyRepresentation struct {
+	Crypto  map[string]interface{} `json:"crypto"` //nolint:revive // This is from Prysm.
+	Version uint                   `json:"version"`
+	Name    string                 `json:"name"`
+}
+
+func EncryptAndStoreKey(key privval.FilePVKey, filePath string) error {
```

Comment:
I suggest we add some unit tests for EncryptAndStoreKey and LoadEncryptedPrivKey functions

Reply:
Good suggestion!

I refactored some codes for test ([60d1b4c](https://github.com/piplabs/story/pull/494/commits/60d1b4c744371f031a8144682eaa52033265737d)) and add some test cases ([5b6711c](https://github.com/piplabs/story/pull/494/commits/5b6711cf988161b507b29c421ae1699093c0502e)).